apps/sr-frontend/.eslintrc.json

```diff
@@ -1,10 +1,12 @@
 {
-  "extends": ["../../.eslintrc.json"],
+  "extends": ["plugin:@nx/react", "../../.eslintrc.json"],
   "ignorePatterns": ["!**/*"],
   "overrides": [
     {
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/consistent-type-imports": "error"
```

Comment:
I think this might be autofixable, not sure if you'd need to add `prefer` or `fixStyle`

Reply:
I tried changing error to prefer and fixStyle, neither worked. unless you mean something else

Reply:
you can use lint to auto fix, but i think we can leave this rule as is for now, looks like bumping eslint when we go to nx17 will prompt us to go through the rules again.

Reply:
both of those settings seem unrelated to what I did. By making this an 'error', vscode will automatically fix it on save if you specify `editor.codeActionsOnSave` to have `source.fixAll: true`

Reply:
should we add those to `.editorconfig`?